packages/cli/src/lib/bundler/config.ts

```diff
@@ -211,8 +275,10 @@ export async function createConfig(
       rules: loaders,
     },
     output: {
+      uniqueName: options.moduleFederation?.name,
```

Comment:
Just a note. This can be used later to enable multiple versions of the same plugin to be loaded.